src/webserve_kq/SocketConnect.cpp

```diff
@@ -79,24 +131,77 @@ void	SocketConnect::printReadData()
 
 }
 
-int			SocketConnect::getSizeR()
+int	SocketConnect::setRequest(std::vector<char> *data_read, std::vector<Server> list_server)
 {
-	return (_sizeR);
-}
+	std::string	str_read = toString(data_read);
+	std::string	oneline;
 
-int			SocketConnect::getSizeW()
-{
-	return (_sizeW);
-}
+	try
```

Comment:
In this try statement, it would be more efficient if we split the extractions blocks and throw customized exceptions. example: try extracting method, if it fails, then throw and exception about that. it makes it easier to debug in case something goes wrong.